src/toolchain-conventions.adoc

```diff
@@ -1,3 +1,17 @@
+== Target Triples
```

Comment:
The Autoconf manual speak of "Triplets" not "Triples" (see https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.72/html_node/Specifying-Target-Triplets.html).

Reply:
Will do.